### DISCUSSION TITLE
Is there any estimation, when bs 5 change into productive status?

### DISCUSSION THREAD

**Discussion by spfeifer222** [twbs/bootstrap#33252] [General]
I know, sometimes is hard to say, but a approximate hint would be nice (e.g. from how long did it takes for bs4), because I start programming and have to decide which version of bs I am using. 

Thanks a lot,
Seb

**Comment by Deckluhm** [comment]
As shown [here](https://github.com/twbs/release), there is no release date yet for v5.

I guess for most projects it's already safe to use v5 (even if it's still in beta, it's already pretty stable and probably won't change much).

You have to choose between using a soon to be outdated version (v4 with jQuery) and migrate later, or use v5 (which probably still is a bit buggy) and gain performances and a generally better version of the lib (and don't need to migrate because this is the latest version).

**Comment by mdo** [comment]
One more beta, then stable most likely. Doing our best—feel free to help tackle issues and PRs if there's something you see you can assist us with :).

**Comment by spfeifer222** [comment]
Thanks a lot!
